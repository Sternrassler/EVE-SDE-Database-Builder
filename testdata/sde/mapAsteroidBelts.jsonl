{"asteroidBeltID":40000201,"solarSystemID":30000001,"asteroidBeltName":"Tanoo I - Asteroid Belt 1","typeID":15,"x":1.5e11,"y":1.0e9,"z":0.0,"celestialIndex":1,"orbitIndex":1,"planetID":40000001}
{"asteroidBeltID":40000202,"solarSystemID":30000142,"asteroidBeltName":"Jita IV - Asteroid Belt 1","typeID":15,"x":2.0e11,"y":1.0e10,"z":-5.0e10,"celestialIndex":4,"orbitIndex":1,"planetID":40000002}
{"asteroidBeltID":40000203,"solarSystemID":30000003,"asteroidBeltName":"H-PA29 III - Asteroid Belt 1","typeID":15,"x":-3.0e11,"y":2.0e11,"z":1.0e11,"celestialIndex":3,"orbitIndex":1,"planetID":40000003}
