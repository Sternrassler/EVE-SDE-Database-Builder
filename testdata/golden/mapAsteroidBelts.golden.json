[
  {
    "asteroidBeltID": 40000201,
    "solarSystemID": 30000001,
    "typeID": 15,
    "x": 150000000000,
    "y": 1000000000,
    "z": 0
  },
  {
    "asteroidBeltID": 40000202,
    "solarSystemID": 30000142,
    "typeID": 15,
    "x": 200000000000,
    "y": 10000000000,
    "z": -50000000000
  },
  {
    "asteroidBeltID": 40000203,
    "solarSystemID": 30000003,
    "typeID": 15,
    "x": -300000000000,
    "y": 200000000000,
    "z": 100000000000
  }
]